### PR TITLE
Add User Experience Design topic

### LINKED ISSUE #2010: Possible new topic: User Experience Design
The LBNL STRUDEL project <https://strudel.science> is starting to do outreach and hold some events.  I want to post a cc for the project as well as events for their upcoming webinar and hackathon.  They focus on user experience design for scientific software.  We could put it under the Better Planning --> Design topic, but it seems like it might be worth opening a distinct topic for UX Design.

The topic could be "User Experience Design" or "User Experience" or "User Experience (UX)" or something like that.  I don't have strong feelings.

### PR DESCRIPTION
**Pull request by bernhold**
Resolves #2010

### CODE REVIEW

File: Articles/WhatIs/WhatIsUserExperienceDesign.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### What is User Experience Design?
+#### Contributed by  [David E. Bernholdt](https://github.com/bernhold)
+#### Publication date: February 20, 2024
+
+<!--deck start--->
+User experience refers to how the user of a piece of software (or other system) interacts with and experiences it.  A thoughtfully designed user experience can make a huge difference to the reception of a piece of software.
+<!--deck end--->
+
+<!--body start--->
+According to the international standard on the ergonomics of human-system interactions (ISO 9341), user experience (UX) is a "user's perceptions and responses that result from the use and/or anticipated use of a system, product, or service."  It is often considered to encompass usability, but also more than that.
```
Comment (markcmiller86) [inline Articles/WhatIs/WhatIsUserExperienceDesign.md:10]: Is the `a` in `...(UX) is a "user's perceptions...` right? I don't think it is.

File: Site/Topics/Planning.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ The ability of two or more software elements to be used together in an applicati
 The ability of software to continue or evolve.  Considerations are often different when viewed from different perspectives (user, developer, manager, funder), but generally relates to resources to maintain or evolve the code, adaptability of the codebase to new uses and new computational platforms.
 <!---Topic order: 6--->
 
+- **User Experience Design**
+An intentional process of designing the user experience for a software (or other) product.
```
Comment (markcmiller86) [inline Site/Topics/Planning.md:38]:
```suggestion
An intentional process of designing for the user experience of a software (or other) product.
```

File: Articles/WhatIs/WhatIsUserExperienceDesign.md (hunk 1 above)
Comment (bernhold) [reply Articles/WhatIs/WhatIsUserExperienceDesign.md:10]: That's a quote from the ISO standard.

Comment (markcmiller86) [reply Articles/WhatIs/WhatIsUserExperienceDesign.md:10]: > That's a quote from the ISO standard.

I am sure it is an important quote but, as written, it didn't parse for me for several mins.

Comment (markcmiller86) [reply Articles/WhatIs/WhatIsUserExperienceDesign.md:10]: Isn't the ISO doc number 9`2`41? 

Here is a proposed revision that I think is clearer.

```suggestion
The international standard on the ergonomics of human-system interactions ([ISO 9241](https://en.wikipedia.org/wiki/ISO_9241)), defines *user experience (UX)* as the "user's perceptions and responses that result from the use and/or anticipated use of a system, product, or service."
It is often considered to encompass usability, but also more than that.
```

**Pull request merged**